### PR TITLE
Update certificate upload component in passiveSTS

### PR DESCRIPTION
**Pull request by savindi7**
passivests-cert

### Related Issues
- https://github.com/wso2/product-is/issues/13488

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/models/application-inbound.ts [hunk 1]
```diff
@@ -327,6 +327,7 @@ export interface WSTrustMetaDataInterface {
 export interface PassiveStsConfigurationInterface {
     realm: string;
     replyTo: string;
+    idToken?: IdTokenConfigurationInterface;
```
Comment (yasinmiran) [inline apps/console/src/features/applications/models/application-inbound.ts:330]: no need `idToken` here

File: apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx [hunk 2]
```diff
@@ -210,146 +197,34 @@ export const InboundPassiveStsForm: FunctionComponent<InboundPassiveStsFormProps
...
+                                <Fragment>
+                                    <Trans
+                                        i18nKey={ "console:develop.features.applications.forms" +
+                                        ".inboundOIDC.sections.certificates.disabledPopup" }
+                                    >
+                                        This certificate is used to encrypt the <Code>id_token</Code>. First, you need
+                                        to disable <Code>id_token</Code> encryption to proceed.
+                                    </Trans>
+                                </Fragment>
+                            ) }
```
Comment (yasinmiran) [inline apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx:215]: Pass an empty fragment here.

```suggestion
                            reasonInsideTooltipWhyDeleteIsNotAllowed={ <Fragment /> }
```

File: apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx [hunk 3]
```diff
@@ -210,146 +197,34 @@ export const InboundPassiveStsForm: FunctionComponent<InboundPassiveStsFormProps
...
-                                    label: t("console:develop.features.applications.forms." +
-                                        "advancedConfig.sections.certificate.fields.type.children.pem.label"),
-                                    value: CertificateTypeInterface.PEM
-                                }
-                            ] }
+                    
+                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                        <ApplicationCertificateWrapper
+                            protocol={ SupportedAuthProtocolTypes.WS_FEDERATION }
+                            deleteAllowed={ !(initialValues?.idToken?.encryption?.enabled) }
```
Comment (yasinmiran) [inline apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx:204]: Delete should be always allowed for passive sts.
```suggestion
                            deleteAllowed={ true }
```

**Review by yasinmiran — CHANGES_REQUESTED** [review]

File: apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx (hunk 3 above)
Comment (savindi7) [reply apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/2977/commits/6d98edaa073d860b55016e33cf784757f1d4a8b2

File: apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx (hunk 2 above)
Comment (savindi7) [reply apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/2977/commits/6d98edaa073d860b55016e33cf784757f1d4a8b2

File: apps/console/src/features/applications/models/application-inbound.ts (hunk 1 above)
Comment (savindi7) [reply apps/console/src/features/applications/models/application-inbound.ts]: Fixed with https://github.com/wso2/identity-apps/pull/2977/commits/6d98edaa073d860b55016e33cf784757f1d4a8b2

**Review by yasinmiran — APPROVED** [review]

File: apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx [hunk 4]
```diff
@@ -74,24 +80,23 @@ export const InboundPassiveStsForm: FunctionComponent<InboundPassiveStsFormProps
...
-     * Set initial PEM values.
+     * Set the certificate type.
      */
     useEffect(() => {
-        if (CertificateTypeInterface.PEM === certificate?.type) {
-            setPEMSelected(true);
-            if (certificate?.value) {
-                setPEMValue(certificate.value);
-            }
+        if (certificate?.type){
```
Comment (Wathsara) [inline apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx:95]:
```suggestion
        if (certificate?.type) {
```

Comment (savindi7) [reply apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx:95]: Fixed with https://github.com/wso2/identity-apps/pull/3122/commits/b8c116aae6984670cb0b4f522f2f86b0ef96028a

**Pull request merged**